data/libraries.json

```
@@ -757,5 +757,14 @@
             "JavaScript"
         ],
         "url": "http://ivan.vucica.net"
+    },
+    {
+        "doap": "https://git.sr.ht/~nicoco/slidge/tree/master/item/doap.xml",
+        "last_renewed": "2023-01-18T00:00:00",
```

```suggestion
        "last_renewed": null,
```

last_renewed is not necessary anymore, since this isn't taken into account after 499b6c79bd9306ececa5f85429f6a2c66c688c2d